tests/PhpBrew/Command/UpdateCommandTest.php

```diff
@@ -14,6 +14,7 @@ class UpdateCommandTest extends CommandTestCase
 
     /**
      * @outputBuffering enabled
+     * @group mayignore
```

Comment:
Add the mayignore group to skip HTTP interaction test during GitHub action running.